airbyte-integrations/connectors/source-klaviyo/pyproject.toml

```diff
@@ -17,7 +17,7 @@ include = "source_klaviyo"
 
 [tool.poetry.dependencies]
 python = "^3.10,<3.12"
-airbyte_cdk = "^6"
+airbyte_cdk = "6.6.8rc13"
```

Comment:
Using predev version temporarily to run CI while [this is merged](https://github.com/airbytehq/airbyte-python-cdk/pull/92), I will use new release [published here](https://github.com/airbytehq/airbyte-python-cdk/releases) once ready.